client.go

```diff
@@ -22,6 +22,9 @@ import (
 	"perun.network/go-perun/wallet"
 )
 
+// Client is the central object to access go-perun.
+// A Client can be used to propose and manage many channels with different
+// peers.
```

Comment:
I think go-perun's description is clearer.
```suggestion
// Client is a state channel client. It is the central controller to interact
// with a state channel network. It can be used to propose channels to other
// channel network peers.
```